en/installation/installation-of-a-central-server/using-packages.md

```diff
@@ -81,36 +120,56 @@ This section describes how to install a Centreon Central server.
 It's possible to install this server with a local database on the server, or
 a remote database on a dedicated server.
 
-<!--DOCUSAURUS_CODE_TABS-->
-
-<!--With a local database-->
-
-Run the commands:
+### With a local database
 
+<!--DOCUSAURUS_CODE_TABS-->
+<!--CentOS 8-->
+```shell
+dnf install -y centreon centreon-database
+systemctl daemon-reload
+systemctl restart mariadb
+```
+<!--CentOS 7-->
+Run the commands for CentOS 7:
 ```shell
 yum install -y centreon centreon-database
 systemctl daemon-reload
 systemctl restart mariadb
 ```
+<!--END_DOCUSAURUS_CODE_TABS-->
 
-<!--With a remote database-->
+### With a remote database
 
 > If installing database on a dedicated server, this server should also have
 > the prerequired repositories.
 
+<!--DOCUSAURUS_CODE_TABS-->
```

Comment:
I think we should split this into 2 double tabs section so the tabs are tied to the commands (ie the text "Run the following..." is outside the tabs).
Rendering is better on Docusorus this way.